index.bs

```diff
@@ -1431,6 +1431,12 @@ Today writing to {{Document/cookie|document.cookie}} in contexts where script-in
 
 Likewise, today reading {{Document/cookie|document.cookie}} in contexts where script-initiated cookie-reading is disallowed typically returns an empty string. However, a cooperating web server can verify that server-initiated cookie-writing and cookie-reading work and report this to the script (which still sees empty string) and the script can use this information to infer that script-initiated cookie-reading is disallowed.
 
+<!-- ============================================================ -->
+## Clear cookies ## {#clear-cookies}
+<!-- ============================================================ -->
+
+When a user clears cookies for an origin, the browser needs to wipe all storage for that origin; including service workers. This is to prevent websites from restoring any user identifiers in persistent storage after a user initiates "clear cookies".
```

Comment:
replace "browser" with "user agent"

maybe reword "a user initiates "clear cookies"" with "a user initiates the action" ?

You may want to look at https://w3c.github.io/webappsec-clear-site-data/#clear-dom for some more precise terminology around "all storage for that origin", although that spec itself still needs to integrate with Storage about what it really means. Specifically, you could write "also clear DOM-accessible storage for that origin"

Reply:
Thanks for the quick review! Sounds good, I added the text about DOM-accessible storage and added a link with the regular mark down linking format. I found one example of this but wasn't sure if its the most appropriate way to link across specs. Let me know if there is a better way.

Reply:
Hmmm... I might leave out the link for now, and/or maybe add an inline `Issue: ....` block talking about how this should link to a canonical definition, maybe in Storage.